File: backend/config/brand.py
"""
HealthNova AI — Backend Brand Configuration

Single source of truth for backend services, emails, health checks,
PDF reports, and administration site branding.
"""

BRAND_NAME = "HealthNova AI"
BRAND_DISPLAY_NAME = "HealthNova AI"
BRAND_SHORT_NAME = "HN"
BRAND_TAGLINE = "AI-Powered Clinical Decision Support & Patient Risk Intelligence"
BRAND_SUBTITLE = BRAND_TAGLINE

BRAND_DESCRIPTION = (
    "HealthNova AI is an intelligent clinical decision-support platform that uses "
    "machine learning, AI, real-time patient information, explainable risk prediction "
    "and clinical intelligence to assist healthcare professionals in making informed decisions."
)

# API Documentation & Metadata
API_TITLE = "HealthNova AI API"
API_DESCRIPTION = "Clinical Decision Support & Patient Risk Intelligence API"
API_VERSION = "1.0.0"

# Django Admin Configuration
ADMIN_SITE_HEADER = "HealthNova AI Administration"
ADMIN_SITE_TITLE = "HealthNova AI"
ADMIN_INDEX_TITLE = "HealthNova AI Administration"

# Role Subtitles
ROLE_SUBTITLES = {
    "DOCTOR": "Clinical Decision Support",
    "NURSE": "Triage & Patient Risk Monitoring",
    "ANALYST": "Clinical Data & Model Intelligence",
    "MEDICAL_INFORMATICIST": "Clinical Data & Model Intelligence",
    "ADMIN": "Platform & Security Administration",
    "IT_ADMIN": "Platform & Security Administration",
    "PATIENT": "Personal Health Intelligence",
}

# Role AI Assistants
AI_ASSISTANTS = {
    "DOCTOR": "HealthNova AI Clinical Assistant",
    "NURSE": "HealthNova AI Triage Assistant",
    "ANALYST": "HealthNova AI Analytics Assistant",
    "MEDICAL_INFORMATICIST": "HealthNova AI Analytics Assistant",
    "ADMIN": "HealthNova AI Platform Assistant",
    "IT_ADMIN": "HealthNova AI Platform Assistant",
    "PATIENT": "HealthNova AI Health Assistant",
}

# Disclaimers
CLINICAL_SAFETY_DISCLAIMER = (
    "HealthNova AI provides clinical decision support and risk insights. "
    "It does not replace professional medical judgment."
)

REPORT_DISCLAIMER = (
    "This report is generated for clinical decision support and should be "
    "interpreted by an appropriately qualified healthcare professional."
)

# Academic Project Lineage
ACADEMIC_PRIMARY_TITLE = (
    "Enhancing Clinical Decision Support Systems Through Patient Risk Level "
    "Prediction Using Machine Learning Techniques"
)
ACADEMIC_SECONDARY_TITLE = (
    "Patient Risk Level Prediction Using Machine Learning for Intelligent "
    "Clinical Decision Support"
)
PROJECT_CODE = "BPY-CSE-2666"

# Contact Channels
SUPPORT_EMAIL = "support@healthnova.ai"
PRIVACY_EMAIL = "privacy@healthnova.ai"
DEFAULT_FROM_EMAIL = "noreply@healthnova.ai"

File: frontend/src/config/brand.ts
/**
 * HealthNova AI — Centralized Frontend Brand Configuration
 *
 * Single source of truth for application branding, display text,
 * subtitles, disclaimers, and version information.
 */

export const BRAND_CONFIG = {
  /** Official brand name */
  brandName: "HealthNova AI",

  /** Official display name */
  displayName: "HealthNova AI",

  /** Compact brand mark / acronym */
  shortName: "HN",

  /** Official product subtitle */
  tagline: "AI-Powered Clinical Decision Support & Patient Risk Intelligence",

  /** Official short description */
  description:
    "HealthNova AI is an intelligent clinical decision-support platform that uses machine learning, AI, real-time patient information, explainable risk prediction and clinical intelligence to assist healthcare professionals in making informed decisions.",

  /** Application software version */
  productVersion: "1.0.0",

  /** Role-specific workspace subtitles */
  roleSubtitles: {
    DOCTOR: "Clinical Decision Support",
    NURSE: "Triage & Patient Risk Monitoring",
    MEDICAL_INFORMATICIST: "Clinical Data & Model Intelligence",
    ANALYST: "Clinical Data & Model Intelligence",
    IT_ADMIN: "Platform & Security Administration",
    ADMIN: "Platform & Security Administration",
    PATIENT: "Personal Health Intelligence",
    USER: "Personal Health Intelligence",
  },

  /** Role-scoped AI Assistant names */
  aiAssistants: {
    DOCTOR: "HealthNova AI Clinical Assistant",
    NURSE: "HealthNova AI Triage Assistant",
    MEDICAL_INFORMATICIST: "HealthNova AI Analytics Assistant",
    ANALYST: "HealthNova AI Analytics Assistant",
    IT_ADMIN: "HealthNova AI Platform Assistant",
    ADMIN: "HealthNova AI Platform Assistant",
    PATIENT: "HealthNova AI Health Assistant",
    USER: "HealthNova AI Health Assistant",
  },

  /** Clinical & legal disclaimers */
  disclaimers: {
    clinicalSafety:
      "HealthNova AI provides clinical decision support and risk insights. It does not replace professional medical judgment.",
    reportDisclaimer:
      "This report is generated for clinical decision support and should be interpreted by an appropriately qualified healthcare professional.",
    humanInTheLoop:
      "HealthNova AI is strictly an assistive clinical decision support tool. It does not independently diagnose, treat, or prescribe.",
  },

  /** Academic project lineage and identifiers */
  academic: {
    primaryTitle:
      "Enhancing Clinical Decision Support Systems Through Patient Risk Level Prediction Using Machine Learning Techniques",
    secondaryTitle:
      "Patient Risk Level Prediction Using Machine Learning for Intelligent Clinical Decision Support",
    projectCode: "BPY-CSE-2666",
  },

  /** Institutional contact channels */
  supportEmail: "support@healthnova.ai",
  privacyEmail: "privacy@healthnova.ai",
} as const;

export type BrandConfig = typeof BRAND_CONFIG;
